POC-OOBE/OOBE-Sandbox/README.md

```diff
@@ -0,0 +1,37 @@
+# PowerToys Out Of Box Experience Proposal and Code
+
+##### Developers: [Furaha Dami�n](https://github.com/furahadamien), [Letitia Kwan](https://github.com/letitiakwan), [Jessica Lim](https://github.com/JessicaLim8)
+##### Program Manager : [Eunice Choi](https://github.com/eunicechoi98)
+##### Designer : Rafael Flora
+
+## Overview
+A team of Microsoft Garage interns have created a proposal for the out-of-box experience after the first time install of the PowerToys app, and a proposal for the UI experience of updates.
+
+## 1. Welcome Screen
+
+#### Rationale:
+PowerToys v17 has a silent launch after first time installation and update. As evidenced in issue #1285, users are left wondering if anything happened at all. The aim of this new feature is to allow users a chance to appreate the install/update of PowerToys while at the same time giving them a chance to explore what PowerToys has to offer if they so wish to. 
+
+#### Design:
+<img align="left" src="./images/welcomeWindow.png" /> Welcome window
+
+
+#### Code:
+The frontend and the backend for this window are in [onLaunchContentDialog.xaml](https://github.com/microsoft/PowerToys/blob/interns/users/t-fudami/documentation/POC-OOBE/OOBE-Sandbox/PowerToys%20Settings%20Sandbox/Views/onLaunchContentDialog.xaml) and [onLaunchContentDialog.xaml.cs](https://github.com/microsoft/PowerToys/blob/interns/users/t-fudami/documentation/POC-OOBE/OOBE-Sandbox/PowerToys%20Settings%20Sandbox/Views/onLaunchContentDialog.xaml.cs) respectively. It is later called in the MainPage as below:
```

Comment:
The links are pointing to the files in the branch `interns/users/t-fudami/documentation`. Once you remove the branch (E.g. after merging the PR), the links won't work.
I would consider linking to the files in the branch `interns/dev-oobe` instead. Up to you!

Reply:
Oh yeah, I re-linked them